util/generate-sample-ledger/tests/bootstrap.rs

```diff
@@ -0,0 +1,50 @@
+use std::{
```

Comment:
Similar to the other test, this should not have to delete stuff that might've been left over from a previous run. It also doesn't do any validation that the binary actually produced anything useful, so personally I am not finding much value here.

Reply:
It is needed to have some kind of integration test in the util generate sample keys crate, otherwise cargo test will not build the binary, and then the other test will fail

Reply:
It's testing that the binary runs without crashing -- we could do more validation, any obvious thing worth testing?

Reply:
I guess we could also merge these crates into one

Reply:
Relying on having a unit test just so that the binary gets built and can be used by a unit test in another crate is exactly what I would call funky...
If we want to do this, the right approach is to move the functionality into a library, and call that through a `dev-dependencies` dependency.

Reply:
i wish that cargo would give me a way to depend on binaries from other crates :(

i really prefer testing the actual binary if i can because the whole point of this is to try to catch issues before waiting for a CD deploy cycle

Reply:
maybe this just shouldn't be a cargo test if cargo can't accomodate this, maybe should build the binaries and then just run a script